metricbeat/module/logstash/node/_meta/test/input.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Could you add the version number to the test json input as we have in the Elasticsearch module?

Reply:
Fixed in 1b1094e.